stable/prometheus-operator/values.yaml

```diff
@@ -196,6 +196,12 @@ alertmanager:
       repository: quay.io/prometheus/alertmanager
       tag: v0.16.1
 
+    ## If true then the user will be responsible to provide a secret with alertmanager configuration
+    ## So when true the config part will be ignored (including templateFiles) and the one in the secret will be used
+    ##
+    useExistingSecret: false
```

Comment:
Could you please also add this configuration to the `ci/` file?

Reply:
Done, but please wait 1-2 more days until I finish an e2e test that I can actually use a secret generated by aws-ssm with this new setting. I only checked pieces so far and I would like to have a test to make sure everything works, so I would not have to submit another PR.
Thanks.